docs/classic-ui/forms.md

```diff
@@ -14,4 +14,90 @@ myst:
 ```{todo}
 Describe how create forms with Plone's default form framework z3c.form.
 Fields, Widgets, Vocabularies aso are descripted in detail in there own chapter and will be referenced frome examples here.
-```
\ No newline at end of file
+```
+
+Plone uses the [z3c.form](http://pythonhosted.org/z3c.form) library to build its forms, via the [plone.z3cform](http://pypi.python.org/pypi/plone.z3cform) integration package.
+
+For an easy way to structure a form and define its widgets and fields, Plone relies on [plone.autoform](http://pypi.python.org/pypi/plone.autoform), in particular its AutoExtensibleForm base class.
+Which is responsible for processing form hints and setting up [z3c.form](http://pythonhosted.org/z3c.form) widgets and groups (fieldsets).
+
+A form, therefore, is simply a view that uses these libraries, although [plone.autoform](http://pypi.python.org/pypi/plone.autoform) provides some helpful base classes that make it easier to construct forms based on the form schema or {ref}`Dexterity behaviors <backend-behaviors-label>`.
+
+
+(classic-ui-forms-general-forms-label)=
+## General forms
+
+The plonecli provides you with an option to add a form to your Plone package.
+
+
+```shell
+cd collective.awesomeaddon
```

Comment:
w/o reading other parts of the documentation nobody has any idea where this comes from.